addons/l10n_hk/data/account_chart_template_data.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+    <record id="l10n_hk_transfer" model="account.account.template">
+        <field name="name">Transfer Account</field>
+        <field name="code">111220</field>
+        <field name="reconcile" eval="True" />
+        <field name="user_type_id" ref="account.data_account_type_current_assets" />
+    </record>
+
+    <!-- Account Chart template -->
+    <record id="l10n_hk_chart_template" model="account.chart.template">
+        <field name="name">Hong Kong - Chart of Accounts</field>
+        <field name="bank_account_code_prefix">112001</field>
```

Comment:
since you specified 6 digits, you'd expect that first bank account gets code 112001, and so the prefix should be 1120

and 1121 for cash